functions/time/gmtime.c

```diff
@@ -1,16 +1,111 @@
 /* gmtime( const time_t * )
-
-   This file is part of the Public Domain C Library (PDCLib).
-   Permission is granted to use, modify, and / or redistribute at will.
+ * From newlib gmtime_r.c
+ * Original Author: Adapted from tzcode maintained by Arthur David Olson.
+ *
+ *  This file is part of the Public Domain C Library (PDCLib).
```

Comment:
pdclib is designed in a way that all platform dependent code can be moved into the platform code. It has one-function per platform.

So either try to upstream this to https://github.com/DevSolar/pdclib in this folder, or move the file to platform/xbox/functions/time/gmtime.c

Going forward I assume we'll keep reverting the `functions/` folder to be vanilla upstream (I think the major roadblock is currently our Makefile, which probably uses wildcards).

Reply:
Done moving the file. I've also deleted the stub : Keeping it makes the final lib use it and thus return NULL which in turn crashes my app (that was the reason that led me doing this date-related stuff).